pages/docs/tutorials/getting-started/request_reply.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Request and reply pattern
+weight: 20
+---
+
+A quite common messaging pattern is [request-reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html), which describes a **requester**, which sends a request message and waits for a reply and a **replier** which receives the request and responds with a reply.
+
+As of AsyncPAI v3, this pattern can be described natively.
+ 
+# Describing a requester
+
+We are gonna use a very simple ping and pong example where a requestor sends the ping and responder respond with pong. Take notice this is an application-level ping and pong and is not related to ping/pong in standards such as WebSocket.
+
+To describe a **requester** in AsyncAPI we make use of an operation that `send`s the ping and expects a `reply` over `pong`.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example with the requester
+  version: 1.0.0
+  description: Simple example with a requester that initiates the request-reply pattern.
+
+channels:
+  ping:
+    address: /
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: /
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+# Describing a replier
+
+To describe a **replier**, we take the same structure as for the requester, with the simple difference, of using the `receive` action instead.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example with replier
+  version: 1.0.0
+  description: Simple example with a replier that replies to the request.
+
+channels:
+  // Same as for the requester
+
+operations:
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      channel: 
+        $ref: '#/channels/pong'
+```
+
+# Sub-patterns in request and reply
```

Comment:
```suggestion
# Sub-patterns in request/reply
```

For consistency, let's stick to `request/reply`, especially since I see you spelling it more frequently this way. 😄✌🏽